NumLib/ODESolver/NonlinearSolver.cpp

```diff
@@ -136,6 +146,7 @@ bool NonlinearSolver<NonlinearSolverTag::Picard>::solve(
 
         // Update x s.t. in the next iteration we will compute the right delta x
         LinAlg::copy(x_new, x);
+        LinAlg::setLocalAccessibleVector(x);
```

Comment:
removed it from NonlinearSolver.cpp, TimeDiscretization.h and UncoupledProcessesTimeLoop.cpp